src/view/global.styl

```diff
@@ -103,9 +103,29 @@ code {
 }
 
 .u-fieldExtraLong {
- width: 740*$px !important;
+ width: 470*$px !important;
```

Comment:
Maybe we could get rid of either `u-fieldLong` or `u-fieldExtraLong`? I just noticed they're only 100px different now so we may not need both. You'll need to test them out.

Reply:
yah, i can see that, there really was not a lot of places each were used, 10 places in total. 
I merged them all to `u-fieldLong` with the 470*px setting.   everything still looks good.